src/PROPOSAL/Propagator.cxx

```diff
@@ -117,10 +117,10 @@ InteractionType Propagator::DoStochasticInteraction(ParticleState& p_cond,
     std::tie(loss_type, comp, loss_energy)
         = utility.EnergyStochasticloss(p_cond.energy, rnd());
 
-    auto stochastic_loss = StochasticLoss((int)loss_type, loss_energy,
+    /*auto stochastic_loss = StochasticLoss((int)loss_type, loss_energy,
```

Comment:
uncomment code

Reply:
I uncommented it because it seems to be unused, I will remove it